### ISSUE TITLE
tqdm is completely broken

### ISSUE THREAD

**Issue by xl0** [Kaggle/docker-python#1443]
## 🐛 Bug

tqdm is very much broken in notebooks.

![image](https://github.com/user-attachments/assets/33b16445-5e8c-4e42-aa79-cb720ab289de)

### To Reproduce

```python
from tqdm.auto import tqdm
from time import sleep

for _ in tqdm(range(10)):
    sleep(0.5)
```

### Expected behavior

Renders a progress bar

**Comment by xl0** [comment]
It definitely used to work: https://www.kaggle.com/code/soheiltehranipour/tqdm-progress-bar-in-python
Here it is rendered, but if you try to run the notebook, you get the "Loading widget...".

**Comment by djherbis** [comment]
Yeah there's been a FE change that has introduced an incompatibility between the BE envs and the FE.
